doc/examples/segmentation/plot_expand_labels.py

```diff
@@ -0,0 +1,54 @@
+"""
+==========================================
+Expand segmentation labels without overlap
+==========================================
+
+When you have a set of regions
```

Comment:
Starts with half a sentence.

Reply:
Somehow that docstring still had the pasted text from a different example. Should be better now.